### PR TITLE
changed: use constructs available in filesystem TS to obtain relative path

### LINKED ISSUE #1513: error when compiling opm-common
I'm using RH7 + devtoolset-7

commit > 3a69bcbf4adbd844935872f1e800bb19b245ae0b ?? Merge pull request #1503 from joakim-hove/wellsegments-alt-cons

compiles without error

next commit 

> 09595b98adb97b79c9147af69a75b492b0c81b56 Merge pull request #1495 from joakim-hove/opmpack-copy-restart

compiles with the following error

/project/multiscale/OPM/tskille/flow_daily_rh7/opm-common/examples/opmpack.cpp: In function 'void copy_file(const std::experimental::filesystem::v1::path&, std::experimental::filesystem::v1::path, const std::experimental::filesystem::v1::path&)':
/project/multiscale/OPM/tskille/flow_daily_rh7/opm-common/examples/opmpack.cpp:79:21: error: 'relative' is not a member of 'fs'
         fname = fs::relative(fname, source_dir);
                     ^~~~~~~~
make[2]: *** [CMakeFiles/opmpack.dir/build.make:63: CMakeFiles/opmpack.dir/examples/opmpack.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2206: CMakeFiles/opmpack.dir/all] Error 2

Other who have the same issue ?

### PR DESCRIPTION
**Pull request by akva2**
fixes build with gcc 7

closes #1513

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: examples/opmpack.cpp [hunk 1]
```diff
@@ -75,8 +75,12 @@ but restart and import files referred to in the deck are also copied. The -o and
 
 
 void copy_file(const fs::path& source_dir, fs::path fname, const fs::path& target_dir) {
-    if (fname.is_absolute())
-        fname = fs::relative(fname, source_dir);
+    if (fname.is_absolute()) {
```
Comment (joakim-hove) [inline examples/opmpack.cpp]: Thanks for fixing. Could maybe leave the `fs::relative()` alternative as a comment?

Comment (akva2) [reply examples/opmpack.cpp]: okay, will do.

**Comment by joakim-hove** [comment]
The code in question is not tested - and not used (yet) :see_no_evil: - so just merge it when you have added the comment.

**Comment by akva2** [comment]
(was) green and approved. merging.

**Pull request merged**